### PR TITLE
Develop task1

### PR DESCRIPTION
**Pull request by RajatMCAF**
Resource Reservation Model

Rajat, Andrii:

We created the class - resource_reservation and added required fields.

We created the tree view, form view and calendar view for resource reservation

We set up the root menu for resource reservation

We created a function inside ResourceReservation class that fetches the current user information and fill "creator" field.

We created the class - resource reservation tag and added required fields

We created the ReservationTag class and added the required fields.

Linkage of resource Tag to Resource Reservation.

Created view for Reservation Tag (tree and form)

We created the class - resource_detail.py and added required fields.

We created the tree view, form view and calendar view for resource reservation

Added view form as button for Resource Detail with functionality of NEW button from Resource Detail (in future NEW will be hide)

Check and refract code using Pylint and Flake

### CODE REVIEW

File: resource_reservation/views/menu.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Add root menu for Resource Reservation module -->
```
Comment (MORPSmcaf) [inline resource_reservation/views/menu.xml:3]: remove all comments, and align attributes 
                   id
                   name
                   action

File: resource_reservation/views/resource_details_views.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+
+        <!-- Define the tree view for the Resource Detail model -->
```
Comment (MORPSmcaf) [inline resource_reservation/views/resource_details_views.xml:5]: remove all comments

File: resource_reservation/views/resource_reservation_tag_views.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Form view configuration for resource reservation tags -->
```
Comment (MORPSmcaf) [inline resource_reservation/views/resource_reservation_tag_views.xml:3]: remove all comments

File: resource_reservation/views/resource_reservation_views.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <!-- Define the view for the Resource Reservation model -->
```
Comment (MORPSmcaf) [inline resource_reservation/views/resource_reservation_views.xml:4]: remove all comments

File: resource_reservation/models/resource_reservation.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        ('other', 'Other'),
+    ],
+        string='Booking Status',
+        default='pending',
+        help="This field represents the status of the booking.")
+    resource_description = fields.Text(string="Reservation Description", required=True)
+    # res_tag = fields.Char(string="Reservation Tag", required=True)
+    resource_tag_id = fields.Many2one('resource.reservation.tag', string="Resource Tag", required=True)
+
+    # Fetches information of current user from odoo environment to our model
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply resource_reservation/models/resource_reservation.py:48]: done

File: resource_reservation/views/menu.xml (hunk 1 above)
Comment (RajatMCAF) [reply resource_reservation/views/menu.xml]: Done

File: resource_reservation/views/resource_details_views.xml (hunk 2 above)
Comment (RajatMCAF) [reply resource_reservation/views/resource_details_views.xml]: Done

File: resource_reservation/views/resource_reservation_tag_views.xml (hunk 3 above)
Comment (RajatMCAF) [reply resource_reservation/views/resource_reservation_tag_views.xml]: Done

File: resource_reservation/views/resource_reservation_views.xml (hunk 4 above)
Comment (RajatMCAF) [reply resource_reservation/views/resource_reservation_views.xml]: Done

**Pull request closed without merging**